extensions/azurecore/package.json

```diff
@@ -164,17 +164,18 @@
 		"hasAzureResourceProviders": true
 	},
 	"dependencies": {
+		"adal-node": "^0.1.28",
```

Comment:
This is to fix packaging issues where webpack didn't detect the transitive dependency?

Reply:
Either it, or one of it's dependencies using expression based require which webpack can't read.